src/components/10-atoms/button-link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.0
+
+- **BREAKING CHANGE**: Same height on all devices. (#1869)
```

Comment:
Minor: `The height of the element at mobile devices changed to a bigger size`

Reply:
I find the message from raphi quite speaking